src/Colore/Native/NativeSdkMethods.cs

```diff
@@ -160,7 +160,7 @@ internal NativeSdkMethods()
             [In] Guid deviceId,
             [In] Effects.Generic.EffectType effectType,
             [In] IntPtr param,
-            [In] [Out] ref Guid effectId);
```

Comment:
The space between attributes should be left intact.
(If this change was automated based on the included code style settings file for ReSharper then that's a miss on my part.)

Reply:
I am using vscode. Default formatter does that. I don't know how can I change that. Right now I am fixing manually. /:

Reply:
Ah I see, yeah I'm not sure if there's something to configure how VSCode formats it.